test/test-server/schema_registry_handlers.go

```diff
@@ -31,15 +31,14 @@ func handleSRUpdateTopLevelConfig(t *testing.T) http.HandlerFunc {
 			err = json.NewEncoder(w).Encode(srsdk.ConfigUpdateRequest{Compatibility: req.Compatibility})
 			require.NoError(t, err)
 		case http.MethodGet:
-			res := srsdk.Config{
-				CompatibilityLevel: "FULL",
-				DefaultMetadata: &srsdk.Metadata{
-					Properties: map[string]string{
-						"owner": "Bob Jones",
-						"email": "bob@acme.com",
-					},
+			res := srsdk.Config{CompatibilityLevel: srsdk.PtrString("FULL")}
```

Comment:
Can we combine this line with line 41?

Reply:
unfortunately no, `Metadata` is not capitalized and I can only use a setter